File: demo/src/main/java/me/shkschneider/skeleton/demo/activity/AboutActivity.java
package me.shkschneider.skeleton.demo.activity;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

import me.shkschneider.skeleton.demo.fragment.AboutFragment;
import me.shkschneider.skeleton.SkeletonFragmentActivity;

public class AboutActivity extends SkeletonFragmentActivity {

    public static Intent getIntent(final Activity activity) {
        return new Intent(activity, AboutActivity.class);
    }

    @Override
    protected void onCreate(final Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        home(true);

        setContentFragment(new AboutFragment());
    }

}

File: demo/src/main/kotlin/me/shkschneider/skeleton/demo/about/AboutActivity.kt
package me.shkschneider.skeleton.demo.about

import android.os.Bundle
import me.shkschneider.skeleton.android.app.SkeletonActivity
import me.shkschneider.skeleton.demo.R

class AboutActivity : SkeletonActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
        toolbar(home = true, title = getString(R.string.title))
    }

    override fun onSupportNavigateUp(): Boolean {
        onBackPressed()
        return true
    }

}
